google/cloud/aiplatform/models.py

```diff
@@ -3210,3 +3211,93 @@ def upload_tensorflow_saved_model(
         )
+
+    def list_model_evaluations(
+        self,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ) -> List["model_evaluation.ModelEvaluation"]:
+        """List all Model Evaluation resources associated with this model.
+
+        Example Usage:
+
+        my_model = Model(
+            model_name="projects/123/locations/us-central1/models/456"
+        )
+
+        my_evaluations = my_model.list_model_evaluations()
+
+        Args:
+            project: Optional[str]=None,
+                Project to get model evaluations from. Overrides project set in
+                aiplatform.init.
+            location: Optional[str]=None,
+                Location to get model evaluations from. Overrides location set in
+                aiplatform.init.
+            credentials: Optional[auth_credentials.Credentials]=None,
+                Custom credentials to get model evaluations from. Overrides credentials
+                set in aiplatform.init.
+        Returns:
+            List[model_evaluation.ModelEvaluation]: List of ModelEvaluation resources
+            for the model.
+        """
+        parent = utils.full_resource_name(
+            resource_name=self.resource_name,
+            resource_noun=Model._resource_noun,
+            parse_resource_name_method=Model._parse_resource_name,
+            format_resource_name_method=Model._format_resource_name,
+            project=project,
+            location=location,
+        )
+
+        self.wait()
+
+        return model_evaluation.ModelEvaluation._list(
+            project=project,
+            location=location,
+            credentials=credentials,
+            parent=parent,
+        )
+
+    def get_model_evaluation(
+        self,
+        evaluation_id: Optional[str] = None,
+    ) -> Optional[model_evaluation.ModelEvaluation]:
+        """Returns a ModelEvaluation resource and instantiates its representation.
+        If no evaluation_id is passed, it will return the first evaluation associated
+        with this model.
+
+        Example usage:
+
+            my_model = Model(
+                model_name="projects/123/locations/us-central1/models/456"
+            )
+
+            my_evaluation = my_model.get_model_evaluation(
+                evaluation_id="789"
+            )
+
+            # If no arguments are passed, this returns the first evaluation for the model
+            my_evaluation = my_model.get_model_evaluation()
+
+        Args:
+            evaluation_id (str):
+                Optional. The ID of the model evaluation to retrieve.
+        Returns:
+            model_evaluation.ModelEvaluation: Instantiated representation of the
+            ModelEvaluation resource.
+        """
+        if not evaluation_id:
+            evaluation_resource_name = self.list_model_evaluations(self.resource_name)[
```

Comment:
Preference to warn if more than one model evaluation is present and log that we are returning this particular model eval resource name.

Additionally, this can just be returned directly.